test/xmltest/__snapshots__/valid.test.js.snap

```diff
@@ -1161,16 +1525,13 @@ Object {
 
 exports[`xmltest/valid standalone should match 115.xml with snapshot 1`] = `
 Object {
-  "actual": "<!DOCTYPE doc><doc>&amp;e1;</doc>",
+  "actual": "<!DOCTYPE doc [
+<!ELEMENT doc (#PCDATA)>
+<!ENTITY e1 \\"&e2;\\">
+<!ENTITY e2 \\"v\\">
+]>
+<doc>&amp;e1;</doc>",
   "errors": Array [
-    Array [
-      "error",
-      "entity not found:&e2;",
-      Object {
-        "columnNumber": 6,
-        "lineNumber": 6,
-      },
-    ],
```

Comment:
Another one: it looks like the sax parser was previously parsing the `&e2;` from the second entity, now it's no longer complaining about it.
Great.

Reply:
Yes, but beware that my implementation makes absolutely no validation on what is inside the internalSubset. It assumes everything between `[ `and `]` is valid as long as all the comments, brackets and quotes are properly closed.

Reply:
Yes, that's good enough for me. I just wanted to highlight the other things it fixes.